settings.gradle

```diff
@@ -1,6 +1,7 @@
 rootProject.name = 'spring-amqp-dist'
 
 include 'spring-amqp'
+include 'spring-amqp-bom'
```

Comment:
Not sure how that works, but I think this module's dir is empty, so that's why it does not appear in the PR content.

Reply:
So, looks like we do just this on GH action:
```
        uses: burrunan/gradle-cache-action@v1
        with:
          arguments: check
```
which really does not involve all the resources with have in the project.

Should we revise it to `build dist` instead to be sure in the docs and all the artifacts we may build in the end?

Reply:
Strsnge - I have `spring-amqp-bom.txt` in the IDE (copied from SI), but I don't see it in the dir; investigating...